vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -1057,32 +1062,54 @@ func resourceVSphereVirtualMachineRead(d *schema.ResourceData, meta interface{})
 		return fmt.Errorf("Invalid disks to set: %#v", disks)
 	}
 
+	// start
 	networkInterfaces := make([]map[string]interface{}, 0)
+
+	deviceList, err := vm.Device(context.TODO())
+	deviceList = deviceList.SelectByType((*types.VirtualEthernetCard)(nil))
+	log.Printf("Device list %+v", mvm.Config.Hardware.Device)
```

Comment:
Can we get the `[DEBUG]` prefix here?

Looks like there are some other entries that need this as well - can we get those there as well?